src/main/kotlin/org/pkl/lsp/ast/Extensions.kt

```diff
@@ -104,12 +104,8 @@ fun PklNode.isAncestor(of: PklNode): Boolean {
   return false
 }
 
-fun PklNode.isInStdlib(): Boolean {
-  return this.enclosingModule?.let { mod ->
-    val uri = mod.uri
-    uri.scheme == "pkl" && uri.authority == "stdlib"
-  } ?: false
-}
+val PklNode.isInStdlib
+  get(): Boolean = this.containingFile is StdlibFile
```

Comment:
Changed this because vals with getters is more in line with Kotlin style (getters should be used for cheap computations)